### PR TITLE
Capture the source file that was requested to be compiled in the symbols

### PR DESCRIPTION
**Pull request by prozacchiwawa**

### CODE REVIEW

File: src/classic/clvm_tools/cmds.rs [hunk 1]
```diff
@@ -669,6 +669,18 @@ pub fn launch_tool(stdout: &mut Stream, args: &[String], tool_name: &str, defaul
...
             .set_help("Only show frames along the exception path".to_string()),
     );
+    parser.add_argument(
+        vec!["-g".to_string(), "--extra-syms".to_string()],
+        Argument::new()
+            .set_action(TArgOptionAction::StoreTrue)
+            .set_help("Produce more diagnostic info in symbols".to_string()),
+    );
+    parser.add_argument(
+        vec!["--symbol-output".to_string()],
```
Comment (aqk) [inline src/classic/clvm_tools/cmds.rs]:
```suggestion
        vec!["--symbol-output-file".to_string()],
```

File: src/classic/clvm_tools/cmds.rs [hunk 2]
```diff
@@ -871,6 +896,17 @@ pub fn launch_tool(stdout: &mut Stream, args: &[String], tool_name: &str, defaul
         }
     }
 
+    let symbol_table_output = parsed_args
+        .get("symbol_output")
```
Comment (aqk) [inline src/classic/clvm_tools/cmds.rs]:
```suggestion
        .get("symbol_output_file")
```

**Review by aqk — DISMISSED** [review]
Great!

File: src/classic/clvm_tools/cmds.rs (hunk 2 above)
Comment (prozacchiwawa) [reply src/classic/clvm_tools/cmds.rs]: good idea

**Review by aqk — APPROVED** [review]

**Pull request merged**